src/doc/html/mod.rs

```diff
@@ -229,9 +229,10 @@ impl<'a> Renderer<'a> {
             for i in &cx.params {
                 write!(
                     out,
-                    "<h3 id=\"parameter.{0}\" class=\"impl\"><code class=\"in-band\">{0}<span class=\"type-annotation\">: {1}</span></code></h3>",
+                    "<h3 id=\"{2}\" class=\"impl\"><code class=\"in-band\"><a href=\"#{2}\">{0}</a><span class=\"type-annotation\">: {1}</span></code></h3>",
```

Comment:
Not directly related, but all those `<h3 ...` strings are very similar and their construction should be refactored IMO.

Reply:
Good point...